windows/runner.ps1

```diff
@@ -64,11 +64,15 @@ param (
 
     [Parameter()]
     [switch]
-    $ProcessStats,
+    $ProcessStats
 
     [Parameter()]
     [double]
-    $CpuUtilizationTimeout = 30
+    $CpuUtilizationTimeout = 30,
+
+    [Parameter()]
+    [switch]
+    $CpuUtilizationOnlyValue
```

Comment:
would it be possible to use the same convention/style as in Unix? `--cpu-utilization-only-value`?

Reply:
PowerShell uses `PascalCase` for public properties. https://poshcode.gitbook.io/powershell-practice-and-style/style-guide/code-layout-and-formatting#capitalization-conventions